cmd/kwil-admin/cmds/setup/testnet.go

```diff
@@ -92,5 +96,8 @@ func testnetCmd() *cobra.Command {
 	cmd.Flags().BoolVar(&withoutNonces, "without-nonces", false, "disable account nonces")
 	cmd.Flags().BoolVar(&withGas, "gas", false, "enable gas")
 	cmd.Flags().Var(&allocs, "alloc", "account=amount pairs of genesis account allocations")
+	cmd.Flags().BoolVar(&snapshotsEnabled, "snapshots.enable", false, "enables db snapshots")
+	cmd.Flags().Uint64Var(&maxSnapshots, "snapshots.max-snapshots", 3, "Maximum number of snapshots to store in the device")
+	cmd.Flags().Uint64Var(&snapshotHeights, "snapshots.heights", 10000, "Heights to snapshot the db")
```

Comment:
Eh, I guess if it's desirable for the flags to match kwild's flags...

Reply:
it will be multiples of these heights. like if the configured height is 10000, we would snapshot at 10k, 20k, 30k etc. I choose this approach so that all the nodes can snapshot deterministically at specific heights. Not sure if there is another good way of specifying this.

Reply:
Yeah, that's totally logical.  The doc here is not clear though.